build-system/server-inabox-template.html

```diff
@@ -22,6 +22,8 @@ <h1>In-a-box Envelope</h1>
   <div style="background: #eee; overflow: hidden;; height: OFFSET;">scroll down to see the ad</div>
 
   <iframe
+      id="inabox-frame"
+      src="AD_URL"
```

Comment:
Duplicate `src` attribute (see below), which might cause errors. I think this line snuck in during the merge.

Reply:
good catch!